packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -23,18 +25,32 @@ type DropdownMenuContextValue = {
   onOpenToggle(): void;
 };
 
+const SubmenuContext = React.createContext<boolean | undefined>(undefined);
 const [DropdownMenuProvider, useDropdownMenuContext] = createContext<DropdownMenuContextValue>(
   DROPDOWN_MENU_NAME
 );
 
-type DropdownMenuOwnProps = {
+type DropdownMenuOwnProps = React.ComponentProps<typeof DropdownMenuImpl>;
+
+const DropdownMenu: React.FC<DropdownMenuOwnProps> = (props) => {
+  const parentSubmenuContext = React.useContext(SubmenuContext);
```

Comment:
Bit of a nit I guess but the `parentSubmenuContext` name here keeps tripping me up... because in my mind a menu is a submenu if it is in another menu, not just when it is inside a submenu. Feels like `parentMenuContext` would be enough?

Reply:
Oh yeah I agree.

Reply:
yes indeedy